content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -64,29 +64,40 @@ spec:
           gpu-vendor.example/example-gpu: 1 # requesting 1 GPU
 ```
 
-## Clusters containing different types of GPUs
+## Automatically labeling nodes with Node Feature Discovery {#node-feature-discovery}
 
-If different nodes in your cluster have different types of GPUs, then you
-can use [Node Labels and Node Selectors](/docs/tasks/configure-pod-container/assign-pods-nodes/)
-to schedule pods to appropriate nodes.
+As and administrator, you can automatically discover and label all your GPU enabled nodes
+by deploying the K8S-Sig project Node Feature Discovery [NFD](https://github.com/kubernetes-sigs/node-feature-discovery).
+NFD enables node feature discovery for Kubernetes. 
+It detects hardware features available on each node in a Kubernetes cluster, and advertises those features using node labels and optionally node extended resources, annotations and node taints. 
+Node Feature Discovery is compatible with any recent version of Kubernetes (v1.21+).
 
-For example:
+Administrators can leverage NFD to also taint nodes with specific features, so that only pods that request those features can be scheduled on those nodes.
```

Comment:
Hrm, my instinct is to remove this sentence.  I liked that the page was basic--just labels.  But, I don't feel strongly about it and defer to SMEs and the doc maintainers.

Reply:
IMHO it's relevant enough to keep here, and adding link to docs is good also.

PS. NFD taint support is still experimental: https://nfd.sigs.k8s.io/usage/customization-guide#node-tainting